engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreEngineActor.scala

```diff
@@ -14,14 +15,23 @@ import cromwell.engine.workflow.workflowstore.WorkflowStoreActor._
 import cromwell.engine.workflow.workflowstore.WorkflowStoreEngineActor.{WorkflowStoreActorState, _}
 import cromwell.services.instrumentation.CromwellInstrumentationScheduler
 import cromwell.util.GracefulShutdownHelper.ShutdownCommand
+import net.ceedubs.ficus.Ficus._
 import org.apache.commons.lang3.exception.ExceptionUtils
 
+import scala.concurrent.duration._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success}
 
 final case class WorkflowStoreEngineActor private(store: WorkflowStore, serviceRegistryActor: ActorRef, abortAllJobsOnTerminate: Boolean)
   extends LoggingFSM[WorkflowStoreActorState, WorkflowStoreActorData] with ActorLogging with WorkflowInstrumentation with CromwellInstrumentationScheduler with WorkflowMetadataHelper {
 
+  val cromwellId: Option[String] = ConfigFactory.load().as[Option[String]]("system.cromwell_id")
+
+  // Entries in the workflow store with a Cromwell ID different from this instance and whose heartbeat timestamps are
+  // older than now - heartbeatTtl are presumed abandoned and up for grabs.
+  val DefaultWorkflowStoreHeartbeatTtl = 1.hour
+  val heartbeatTtl: Duration = ConfigFactory.load().as[Option[Duration]]("system.workflow_heartbeat_ttl").getOrElse(DefaultWorkflowStoreHeartbeatTtl)
```

Comment:
```scala
val heartbeatTtl: FiniteDuration = ConfigFactory.load()
  .getOrElse("system.workflow_heartbeat_ttl", DefaultWorkflowStoreHeartbeatTtl)
```

Also very ToL: Embedded use of `ConfigFactory.load()` works (the load is even cached by ConfigFactory so it's pretty efficient), but makes it hard(er) for specs to pass in their own test configuration. For the sub-project running the test it requires a custom `src/test/resources/application.conf`. In a lot of places I thought we were passing around global and "local" `Config` objects, but I don't see those around here. Food for future thought.